pdns/dnsdistdist/dnsdist-svc.cc

```diff
@@ -91,3 +91,35 @@ bool generateSVCPayload(std::vector<uint8_t>& payload, const SVCRecordParameters
 {
   return generateSVCPayload(payload, parameters.priority, parameters.target, parameters.mandatoryParams, parameters.alpns, parameters.noDefaultAlpn, parameters.port, parameters.ech, parameters.ipv4hints, parameters.ipv6hints, parameters.additionalParams);
 }
+
+struct SVCRecordParameters parseSVCParameters(const svcParamsLua_t& params) {
+  struct SVCRecordParameters parameters;
+  for (const auto& p : params) {
+    if (p.first == "mandatory") {
+      for (auto const &entry : boost::get<std::vector<std::pair<int, std::string>>  >(p.second)) {
+        parameters.mandatoryParams.insert(SvcParam::keyFromString(entry.second));
+      }
+    } else if (p.first == "alpn") {
+      for (auto const &entry : boost::get<std::vector<std::pair<int, std::string> > >(p.second)) {
+        parameters.alpns.push_back(entry.second);
+      }
+    } else if (p.first == "no_default_alpn") {
```

Comment:
Perhaps `noDefaultAlpn` would be more consistent? I don't care that much, to be clear.
```suggestion
    } else if (p.first == "noDefaultAlpn") {
```